crates/rome_js_analyze/src/analyzers/nursery/use_simple_number_keys.rs

```diff
@@ -0,0 +1,273 @@
+use crate::JsRuleAction;
+use rome_analyze::{context::RuleContext, declare_rule, ActionCategory, Ast, Rule, RuleDiagnostic};
+use rome_console::markup;
+use rome_diagnostics::Applicability;
+use rome_js_factory::make;
+use rome_js_syntax::{JsLiteralMemberName, JsSyntaxKind, JsSyntaxToken};
+use rome_rowan::{AstNode, BatchMutationExt};
+use std::str::FromStr;
+
+declare_rule! {
+    /// Disallow number literal object member names which are not base10 or uses underscore as separator
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```js,expect_diagnostic
+    /// ({ 0x1: 1 });
+    /// ({ 11_1.11: "ee" });
+    /// ({ 0o1: 1 });
+    /// ({ 1n: 1 });
+    /// ({ 11_1.11: "ee" });
+    /// ```
```

Comment:
```suggestion
    /// ```js,expect_diagnostic
    /// ({ 0x1: 1 });
    /// ```
    /// ```js,expect_diagnostic
    /// ({ 11_1.11: "ee" });
	/// ```
    /// ```js,expect_diagnostic
    /// ({ 0o1: 1 });
	/// ```
    /// ```js,expect_diagnostic
    /// ({ 1n: 1 });
	/// ```
    /// ```js,expect_diagnostic
    /// ({ 11_1.11: "ee" });
    /// ```
```

@dhrjarun that's how the error cases should be written (please check the other rules too). Then run `cargo lintdoc` and the documentation should be generated.

Reply:
Thanks, @ematipico, Now I understand it.